gapir/client/session.go

```diff
@@ -327,6 +328,17 @@ func (s *session) newADB(ctx context.Context, d adb.Device, abi *device.ABI, lau
 	}
 
 	log.I(ctx, "Launching GAPIR...")
+	// Configure GAPIR to be traceable for replay profiling
+	cleanup, err := perfetto_android.SetupProfileLayers(ctx, d, apk.Name, true, abi, []string{})
```

Comment:
This makes me nervous. This means no matter WHAT we are doing the profiling layers will be turned on in GAPIR. That seems like something we do not want. I do not trust that the profiling layers will be .. stable enough .. to handle all of the odd stuff we do in GAPID replay that ISN'T just "replay this frame".

Reply:
Would the solution be to add the profiling layers using a transform like the FindIssue one which adds the validation layers?
See https://github.com/google/gapid/blob/1e1473feeb407daa5ddfa7491b0dc1032b70a8d9/gapis/api/vulkan/find_issues.go#L85

Reply:
I've added an additional commit that does the layer loading explicitly. It still requires some config, but at least gapir will not load the stages layer when not in a replay.